README.md

```diff
@@ -1,7 +1,10 @@
-## Getting Started
+# Diesel 
+Diesel is a Safe, Extensible ORM and Query Builder for Rust.
+Diesel is the most productive way to interact with databases in Rust because of its safe and composable abstractions over queries.
```

Comment:
This repository isn't Diesel itself but its website. So this may cause confusion.